rotkehlchen/externalapis/etherscan.py

```diff
@@ -532,9 +534,9 @@ def get_token_transaction_hashes(
         yield _hashes_tuple_to_list(hashes)
 
     def has_activity(self, account: ChecksumEvmAddress) -> EtherscanHasChainActivity:
-        """Queries transactions, internal_txs and tokentx for an address with limit=1
+        """Queries balance, transactions, internal_txs and tokentx for an address with limit=1
         just to quickly determine if the account has had any activity in the chain.
-        We make a distinction between transactions and ERC20 transfers since ERC20
+        We make a distinction between balance, transactions and ERC20 transfers since ERC20
```

Comment:
why mention balance here?

```suggestion
        We make a distinction between transactions and ERC20 transfers since ERC20
```